### PR TITLE
Ayomiku cu #860r2xdku handle added sugars and attached json data for each generated page

### PR DESCRIPTION
**Pull request by Ayomikuolatunji**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,6 +100,12 @@ in order to merge to `main`. We prefer to merge to `main` (hence a release) as s
    updated storybook components. This can be of tremendous help during the code review.
 5. Once you are done with the work, assign your PR to a reviewer.
 
+```
```
Comment (hhimanshu) [inline README.md:105]: Can you give a heading about this code and explain what this is?

File: README.md [hunk 2]
```diff
@@ -100,6 +100,12 @@ in order to merge to `main`. We prefer to merge to `main` (hence a release) as s
...
+var element = document.getElementById('ff-page-slug');
```
Comment (hhimanshu) [inline README.md]: use `let`

File: src/__test__/utils.test.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { getAgeRangesForLifeStage, getPercentDaily, spaceToDashes } from "../helpers/utils";
+// import { render } from "@testing-library/react";
+import { RdiAge } from "../models";
+
+describe("Utils Testing", () => {
```
Comment (hhimanshu) [inline src/__test__/utils.test.tsx]: Move test file to where the source file is.

File: src/templates/FoodDetails.tsx [hunk 4]
```diff
@@ -171,8 +172,15 @@ const FoodDetails: React.FC<Props> = ({ pageContext: { recommendedDailyIntakes,
     setDefaultSelectedAgeForGender(selectedLifeStage, setSelectedAge);
   }, [selectedLifeStage, setSelectedAge]);
 
+  console.log(slug);
```
Comment (hhimanshu) [inline src/templates/FoodDetails.tsx]: Remove `console.log()`

File: src/templates/FoodDetails.tsx [hunk 5]
```diff
@@ -171,8 +172,15 @@ const FoodDetails: React.FC<Props> = ({ pageContext: { recommendedDailyIntakes,
...
+
   return (
-    <div>
+    <div
+      id={`ff-page-${slug}`}
+      data-json={JSON.stringify(nutritionFacts)}
+      age-json={JSON.stringify(ageRanges)}
```
Comment (hhimanshu) [inline src/templates/FoodDetails.tsx]: Remove `age-json`. This is not adding much value

File: src/templates/FoodDetails.tsx [hunk 6]
```diff
@@ -171,8 +172,15 @@ const FoodDetails: React.FC<Props> = ({ pageContext: { recommendedDailyIntakes,
...
+      selected-gender={selectedLifeStage}
```
Comment (hhimanshu) [inline src/templates/FoodDetails.tsx]: Remove `selected-gender`, not adding much value

File: src/templates/FoodDetails.tsx (hunk 4 above)
Comment (Ayomikuolatunji) [reply src/templates/FoodDetails.tsx]: done

File: src/templates/FoodDetails.tsx (hunk 5 above)
Comment (Ayomikuolatunji) [reply src/templates/FoodDetails.tsx]: Done

File: src/templates/FoodDetails.tsx (hunk 6 above)
Comment (Ayomikuolatunji) [reply src/templates/FoodDetails.tsx]: Done

File: src/__test__/utils.test.tsx (hunk 3 above)
Comment (Ayomikuolatunji) [reply src/__test__/utils.test.tsx]: done

**Review by hhimanshu — APPROVED** [review]

**Pull request merged**